app/liquid_tags/user_subscription_tag.rb

```diff
@@ -75,14 +89,16 @@ class UserSubscriptionTag < LiquidTagBase
       profileImages.forEach(function(profileImage) {
         profileImage.src = subscriber.profile_image_90;
         profileImage.alt = `${subscriber.username} profile image`;
-        profileImage.style.display = 'block';
       });
+    }
 
-      const profileImageWrappers = document.querySelectorAll(`span${identifier}`);
-
-      profileImageWrappers.forEach(function(profileImageWrapper) {
-        profileImageWrapper.style.display = 'inline-block';
-      });
+    if (isUserSignedIn()) {
+      profileImageContainer.classList.remove("signed-out");
+      profileImageContainer.classList.add("signed-in");
+    } else {
+      profileImageContainer.classList.remove("signed-in");
+      profileImageContainer.classList.add("signed-out");
+      subscriberImageContainer.classList.add("hidden");
```

Comment:
This logic is handled further down with https://github.com/thepracticaldev/dev.to/blob/c4cd91f8102a076004eb0365b0ba41c853b73a68/app/liquid_tags/user_subscription_tag.rb#L219-L231

It looks like you already updated the methods referenced there as well (i.e. `showSignedOut()`) 🎉

Reply:
I refactored this in the latest commit: https://github.com/thepracticaldev/dev.to/pull/9003/commits/44d85d2a773e2064cdf9cd2ab8cd1d7634c21e90#diff-dbaa157d5eab1612b5396d10b80fe577